charts/internal/openstack-infra/templates/main.tf

```diff
@@ -1,8 +1,8 @@
-provider "openstack" {
-  auth_url    = "{{ required "openstack.authURL is required" .Values.openstack.authURL }}"
-  domain_name = "{{ required "openstack.domainName is required" .Values.openstack.domainName }}"
-  tenant_name = "{{ required "openstack.tenantName is required" .Values.openstack.tenantName }}"
-  region      = "{{ required "openstack.region is required" .Values.openstack.region }}"
+provider openstack {
+  auth_url    = {{ required "openstack.authURL is required" .Values.openstack.authURL }}
```

Comment:
I am curious how you tested your changes locally.

With this PR:

```
$ k logs seed-ccee.infra.tf-apply-rhdgq

There are some problems with the configuration, described below.

The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.

Error: Missing newline after argument

  on tf/main.tf line 2, in provider "openstack":
   2:   auth_url    = https://identity-3.eu-nl-1.cloud.sap/v3/

An argument definition must end with a newline.


Error: Invalid number literal

  on tf/main.tf line 36, in resource "openstack_networking_subnet_v2" "cluster":
  36:   cidr            = 10.250.0.0/16

Failed to recognize the value of this number literal.

Thu Jun 25 16:12:54 UTC 2020 terraform.sh exited with 1.
```